src/main/java/org/apache/pirk/querier/wideskies/QuerierDriver.java

```diff
@@ -91,7 +91,7 @@ public static void main(String... args) throws IOException, InterruptedException
     int dataPartitionBitSize = 0;
     int paillierBitSize = 0;
     int certainty = 0;
-    String queryName = null;
```

Comment:
The queryName corresponds to the schemaName in the query-schema.xsd file *not* to the user-given identifier for the  query -- the identifier for this query is actually the queryNum. Notice that queryNum is currently a double -- if we want to start appending dates or whatever else to it, we need to change its type to String throughout.